src/Our.Umbraco.StackedContent/Web/Helpers/ViewHelper.cs

```diff
@@ -20,7 +20,7 @@ private class DummyController : Controller { }
             }
         };
 
-        public static void AddViewLocations(params string[] viewLocations)
+        public static void AddViewLocationFormats(params string[] viewLocations)
```

Comment:
Just to be pedantic, can you rename the param aswell `viewLocations`. Sorry :D

Reply:
Haha of course 😆